fe/fe-core/src/main/java/com/starrocks/backup/Repository.java

```diff
@@ -169,6 +169,15 @@ public long getCreateTime() {
 
     // create repository dir and repo info file
     public Status initRepository() {
+        Status st = initRepository_internal("__palo_repository_");
+        if (st.ok()) {
+            return Status.OK;
+        }
+        return initRepository_internal("__starrocks_repository_");
+    }
+
+    public Status initRepository_internal(String path) {
```

Comment:
Should follow the Java name standard.
```suggestion
    public Status initRepositoryInternal(String path) {
```

Reply:
have fix it